test/val/val_limits_test.cpp

```diff
@@ -320,3 +321,71 @@ TEST_F(ValidateLimits, StructNestingDepthBad) {
       HasSubstr(
           "Structure Nesting Depth may not be larger than 255. Found 256."));
 }
+
+// clang-format off
```

Comment:
Turning off formatting for a whole function seems ... excessive.  What are you really trying to protect?